docs/maps/connect-to-ems.asciidoc

```diff
@@ -69,7 +67,7 @@ docker run --rm --init --publish 8080:8080 \
   {ems-docker-image}
 ----------------------------------
 
-Once {hosted-ems} is running, follow instructions from the webpage at `localhost:8080` to define a configuration file and download the basemaps database.
+Once {hosted-ems} is running, follow instructions from the webpage at `localhost:8080` to define a configuration file and optionally download a more detailed basemaps database.
```

Comment:
I don't see this content in the preview of the doc.

Reply:
There's a conditional for this part to show up for released version builds, I don't know if there's a way to see this in a preview 😓